service/src/main/java/bio/terra/cbas/controllers/RunSetsApiController.java

```diff
@@ -147,6 +152,16 @@ public ResponseEntity<RunSetListResponse> getRunSets(UUID methodId, Integer page
     return new ResponseEntity<>(response, HttpStatus.OK);
   }
 
+  public void abortRequestTimerSample(Timer.Sample requestTimerSample) {
+    micrometerMetrics.stopTimer(
+        requestTimerSample,
+        "cromwell_request_to_initial_submission_timer",
+        "run_set_id",
+        "",
+        "requestSuccessful",
+        "false");
```

Comment:
Are these what show up in `customDimensions` in the Azure Portal? 🤔

Reply:
yes!

Reply:
After discussing: call this something else, don't set run set ID as empty string